samples/boards/altera_max10/sysid/README.rst

```diff
@@ -0,0 +1,34 @@
+.. _altera_max10_sysid-sample:
+
+Altera Nios-II System ID sample
+###############################
+
+Overview
+********
+This sample application demonstrates the usage of Nios-II
+System ID soft IP on Altera MAX10 board.
+
+Requirements
+************
+
+This application uses Altera MAX10 development board for the demo.
+
+Building
+********
+
+.. zephyr-app-commands::
+   :zephyr-app: samples/boards/altera_max10/sysid
+   :board: altera_max10
+   :goals: build
+   :compact:
+
+Sample Output
+=============
+
+nios2 core output
+---------------
```

Comment:
underline needs to be at least as long as the title (add some more ----s)

Reply:
Ok.